generate_openj9_scc.sh

```diff
@@ -101,7 +104,7 @@ function dry_run() {
 
     java -Xshareclasses:name=dry_run_scc,cacheDir=/opt/java/.scc,destroy || true
 
-    SCC_SIZE="${SCC_SIZE:0:-1}"
+    SCC_SIZE="$(printf '%s' "SCC_SIZE" | cut -c 1-$((${#SCC_SIZE}-1)))"
```

Comment:
This is pretty ugly, so if anyone knows a better way to do this feel free to let me know.